### PR TITLE
fix(ci): fixed download-artifacts for kernel testing matrixes.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area CI

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/pages.yml [hunk 1]
```diff
@@ -29,16 +29,17 @@ jobs:
...
       - name: Download matrix ARM64
         uses: actions/download-artifact@v3
         with:
-          name: matrix_master_ARM64
+          name: matrix_master_ARM64.md
       
       - name: Disable Table Of Content for matrixes pages
         run: |
-          mv matrix_*/*.md docs/
+          mv matrix_master_X64.md docs/matrix_X64.md
```
Comment (FedeDP) [inline .github/workflows/pages.yml:41]: Move the downloaded artifacts under the expected name in docs/ folder.

File: .github/workflows/reusable_kernel_tests.yaml [hunk 2]
```diff
@@ -90,7 +90,7 @@ jobs:
         
       - uses: actions/upload-artifact@v3
         with:
-          name: matrix_${{ inputs.libsversion }}_${{matrix.architecture}}
+          name: matrix_${{ inputs.libsversion }}_${{matrix.architecture}}.md
```
Comment (FedeDP) [inline .github/workflows/reusable_kernel_tests.yaml:93]: Upload artifacts with `md` extension

File: .github/workflows/release-body-driver.yml [hunk 3]
```diff
@@ -95,7 +95,8 @@ jobs:
       # Finally, merge them together
       - name: Append matrixes to create release body
         run: |
-          mv matrix_*/*.md .
+          mv matrix_${{ github.event.release.tag_name }}_X64.md matrix_X64.md
```
Comment (FedeDP) [inline .github/workflows/release-body-driver.yml:98]: Rename them. This is just to avoid using the long name.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Comment by FedeDP** [comment]
https://imgflip.com/i/81e4ob

**Review by leogr — APPROVED** [review]

**Pull request merged**